src/platforms/react-native/troubleshooting.mdx

```diff
@@ -242,3 +242,15 @@ React Native 0.60.3 tooling was unexpectedly moving source maps files, which cau
 ## Missing Java stacktrace on the New Architecture
 
 If you're using the new architecture on React Native, you might be missing the Java stacktrace. Follow the [issue](https://github.com/facebook/react-native/issues/34923) for more information.
+
+## Failing Build Due to `sentry-cli`
+
+If you're building your app without access to [sentry.io](https://sentry.io) or some other unexpected error occurs, you can allow failure of the `sentry-cli` step by adding an environmental variable `SENTRY_ALLOW_FAILURE` equal to `true`. The CLI will still run, but won't cause the build to fail:
+
+```bash
+# iOS
+SENTRY_ALLOW_FAILURE=true npx react-native run-ios
+
+# Android
+SENTRY_ALLOW_FAILURE=true npx react-native run-android
```

Comment:
I'd jump a line after exporting the env. var.

Reply:
I created the example without exporting the ENV and instead ran the single build command with the ENV.

Like this, it's less likely that people will forget about it. Also, it's not persistent, so there won't be trouble with forgotten FAILURE=true.

Reply:
mmm that makes a lot of sense, lets keep then.